### ISSUE TITLE
What validation set was used?

### ISSUE THREAD

**Issue by buhuangliu** [ialhashim/DenseDepth#54]
I saw that in the keras implementation, final eval data was used as validation set.  Did you guys use a different validation set other than final eval data during training? Or the final eval data was used as validation set?

I tried naively splitting the 50k training images into train and validation set and this does not work as validation set comes from a distribution too close to training set.

**Comment by ialhashim** [comment]
What do you mean it does not work?

**Comment by buhuangliu** [comment]
I ended up over-fitting the validation set. Both validation set and training set gives a way lower loss than test set. 

Did you use any specific strategies to divide the training data?

**Comment by ialhashim** [comment]
Since we are doing supervised learning this is will always end up happening unfortunately. What I did was similar to what you did, I also tried the model on additional synthetic data and other datasets and kept modifying it some more. For the final tuning of the hyper parameters I started looking at how the test set for the different datasets is affected. This always shows counter intuitive results when tracking the training and validation set and testing. But what it does help is for finding good learning rate or batch size that do not result in quicker overfitting.

So bottom line is try different datasets (e.g. scannet). Or try the larger NYUv2 dataset that would be around 300K images. Tune your model to be least over-fitting then test it.
